100_essentials/greetings/README.md

```diff
@@ -0,0 +1,59 @@
+# Greetings
+
+Everything you need to complete this challenge can be found [here](https://docs.python.org/3.8/tutorial/controlflow.html#defining-functions).
+
+---
+
+Define a function named `greet` that takes in a name and a message as inputs and returns a greeting in the following format:
+
+```python
+>>> greet("Mark", "Oh, hi")
+'Oh, hi, Mark!'
+```
```

Comment:
Just to ensure that I understand why the parameters / arguments are in that specific order, you're trying to differentiate the greeting function from something like `print`, yeah? Potentially also to provide them a slight curveball in ensuring their arguments are in the right order?

Not a huge deal either way, I just like to double check that if we're having them call a function in a semi-unreadable way (e.g. with the parameters "backwards") that there's a solid reason we can stand behind.